src/calculateRentalCost.js

```diff
@@ -4,7 +4,22 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+  const shortTerm = 3;
+  const longTerm = 7;
+  const shortTermDisc = 20;
+  const longTermDisc = 50;
+  const noDisc = price * days;
```

Comment:
we don't usually use use shortcuts for names, so let's rename it to something like `totalCost`
And just a heads-up that it's a good practice in JavaScript to use SNAKE_CASE (all uppercase) for constant variables.
![image](https://github.com/mate-academy/js_task-transportation-on-vacation/assets/46566190/17912edd-4fa4-41ed-858e-8ed98ec12c10)